src/corporacreator/preprocessors/et.py

```diff
@@ -6,5 +6,18 @@ def et(client_id, sentence):
     Returns:
       (str): Cleaned up sentence. Returning None or a `str` of whitespace flags the sentence as invalid.
     """
-    # TODO: Clean up en data
+    # this if loop will skip the following else loop in the case that
+    # the sentence is purely Estonian characters as per https://github.com/
+    # unicode-cldr/cldr-misc-full/blob/master/main/et/characters.json
+    if set(list(sentence)).issubset(set([' ', 'a', 'b', 'c', 'd', 'e', 'f', 'g',
```

Comment:
I'd move the cahracter-`set` outside of the function, for readability, and possibly perf (i.e. initialize that set only once, not sure how clever the python interpreter is).